### PR TITLE
update Az.Accounts version from 2.2.4 to 2.2.8

### PR DESCRIPTION
**Pull request by TheOnlyWei**
## Description

update Az.Accounts version from 2.2.4 to 2.2.8

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/Az.Accounts.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.2.4'
```
Comment (bganapa) [inline src/Accounts/Accounts/Az.Accounts.psd1:15]: this version of module is not used, could you pls revert this change

File: src/Billing/Billing/Az.Billing.psd1 [hunk 2]
```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.4'; })
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.8'; })
```
Comment (bganapa) [inline src/Billing/Billing/Az.Billing.psd1:56]: ModuleVersion -> RequiredVersion

File: src/Accounts/Accounts/Az.Accounts.psd1 (hunk 1 above)
Comment (TheOnlyWei) [reply src/Accounts/Accounts/Az.Accounts.psd1:15]: Hi, do you mean revert 2.2.8 to 2.2.4?

**Review by bganapa — APPROVED** [review]
